doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-test-scenario-export-spreadsheet-proc.adoc

```diff
@@ -0,0 +1,10 @@
+[id='test-designer-test-scenario-export-spreadsheet-proc']
+= Exporting a test scenario spreadsheet
+
+Follow the procedure below to export a test scenarios spreadsheet data using the Test Scenario designer.
```

Comment:
Are we exporting a test scenario spreadsheet or the test scenario's spreadsheet data?

s/a test scenarios spreadsheet data/a test scenario's spreadsheet data
or
s/a test scenarios spreadsheet data/a test scenario spreadsheet

If it is only the data that can be imported and exported, then perhaps the wording should differentiate between the test scenario spreadsheet in the Test Scenario designer and the data of the spreadsheet?

Reply:
@csherrar, We are exporting test scenario's spreadsheet data.

Reply:
what is spreadsheet data? Are you talking about the metadata of the spreadsheet itself or data contained in a spreadsheet? remember that a .csv file is not a spreadsheet. It a values file that contains entries separated by commas.